packages/extract/test/Extract.tests.ts

```diff
@@ -25,7 +26,7 @@ describe('Extract', () =>
     {
         const renderer = new Renderer();
         const sprite = new Sprite(Texture.WHITE);
-        const extract = renderer.plugins.extract as Extract;
+        const extract = renderer.extract as Extract;
```

Comment:
Testing locally I don't think these need to be cast anymore

```suggestion
        const extract = renderer.extract;
```